src/System.IO.FileSystem/tests/FileStream/ctor_str_fm.cs

```diff
@@ -38,6 +38,23 @@ public void InvalidModeThrows()
             AssertExtensions.Throws<ArgumentOutOfRangeException>("mode", () => CreateFileStream(GetTestFilePath(), ~FileMode.Open));
         }
 
+        [Theory, MemberData(nameof(TrailingCharacters))]
+        public void MissingFile_ThrowsFileNotFound(char trailingChar)
+        {
+            string path = GetTestFilePath() + trailingChar;
+            Assert.Throws<FileNotFoundException>(() => CreateFileStream(path, FileMode.Open));
+        }
+
+
+        // Issue #19965
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
I'll fix this after I get results- forgot about ActiveIssue taking TestPlatforms.